auto_round/compressors/diffusion/README.md

```diff
@@ -0,0 +1,87 @@
+# AutoRound for Diffusion models
+
+This feature is experimental and may be subject to changes, including potential bug fixes, API modifications, or adjustments to default parameters.
+
+## Quantization
+
+### API Usage (CPU/GPU) Recommended
+
+By default, AutoRoundDiffusion only quantizes the transformer module of diffusion models and uses `COCO2014 captions` for calibration.
+
+```python
+from auto_round import AutoRoundDiffusion
+from diffusers import AutoPipelineForText2Image
+
+## load the model
```

Comment:
better follow google style, single # and upper case

Reply:
updated